pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -123,6 +124,10 @@ func Register(ctx context.Context, workload *config.UserContext) {
 
 	workload.Core.Namespaces("").AddLifecycle(ctx, "namespace-auth", newNamespaceLifecycle(r, sync))
 	management.Management.RoleTemplates("").AddHandler(ctx, "cluster-roletemplate-sync", newRTLifecycle(r))
+
+	// If a CRTB that is owned by a GRB is updated or deleted re-enqueue the GRB to reconcile the modified CRTB.
```

Comment:
Is there any chance that there is a CRTB owned by a GRB that isn't this particular restricted-admin-cluster-owner one? Do we want this to apply the same way in those cases?

Reply:
Not right now, but I believe we would still want this logic if there were.